pages/api/posts/[postId]/comments/index.js

```diff
@@ -0,0 +1,41 @@
+import { Record, String, Optional, Union, Literal, ValidationError } from 'runtypes'
+import { Comment } from '../../../../../models/comment'
+import connectDB from '../../../../../middleware/mongodb'
+
+const createCommentRunType = Record({
+    query: Record({
+        postId: String,
+    }),
+    body: Record({
+        body: String,
+        replyTo: Optional(String),
+        replyCount: Optional(Number)
```

Comment:
this shouldnt be passed in at creation, because it will only have a value above zero after someone replies to the comment. it should just be 0 by default on creation.